mkosi/config.py

```diff
@@ -4337,6 +4344,7 @@ def summary(config: Config) -> str:
                          Passphrase: {none_to_none(config.passphrase)}
                            Checksum: {yes_no(config.checksum)}
                                Sign: {yes_no(config.sign)}
+                       OpenPGP Tool: ({"gpg" if config.openpgp_tool is None else config.openpgp_tool})
```

Comment:
This can't really be `None`, since `gpg` is the default. The worst thing would be, that it's empty
```suggestion
                       OpenPGP Tool: ({config.openpgp_tool or "gpg"})
```